### ISSUE TITLE
TridentTopolgy

### ISSUE THREAD

**Issue by bigdatacamp** [hdinsight/hdinsight-storm-examples#15]
is posible créate a Trident topology wiht SCP .NET and C#? If yes, any sample?

Thanks

**Comment by BennyM** [comment]
I don't think so. There's no reference to in the SCP library. The docs (although not quite complete) also mention Trident only with Java.Mentioned here: https://github.com/Azure/azure-content/blob/master/articles/hdinsight/hdinsight-storm-overview.md

**Comment by rtandonmsft** [comment]
@bigdatacamp No, it is not possible to use Trident with SCP.Net today.

SCP.Net however supports transactional topologies just like Storm, which you can use for exactly once semantics: https://github.com/hdinsight/hdinsight-storm-examples/tree/master/SCPNetExamples/HelloWorldTx
